util/helm/client.go

```diff
@@ -293,6 +318,7 @@ func newTLSConfig(creds Creds) (*tls.Config, error) {
 
 // Normalize a chart name for file system use, that is, if chart name is foo/bar/baz, returns the last component as chart name.
 func normalizeChartName(chart string) string {
+	strings.Join(strings.Split(chart, "/"), "_")
```

Comment:
@alexmt The return value of `strings.Join()` is not used, is this a bug, or unnecessary code?
/cc @jannfis